packages/vscode-extension/src/handlers.ts

```diff
@@ -1644,22 +1644,8 @@ export async function openReadMeHandler(args: any[]) {
 
 export async function promptSPFxUpgrade() {
   if (globalVariables.isSPFxProject) {
-    let projectSPFxVersion = null;
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion, @typescript-eslint/no-non-null-asserted-optional-chain
-    const yoInfoPath = path.join(globalVariables.workspaceUri?.fsPath!, "SPFx", ".yo-rc.json");
```

Comment:
Any compatibility issue if removing this? E.g., opening old project?

Reply:
The logic doesn't change, I just moved it to util function so prerequisite checker can reuse.